### ISSUE TITLE
Clustering parameters inconsistencies

### ISSUE THREAD

**Issue by pdec** [gtonkinhill/panaroo#102]
Hey,

you've done a great job with Panaroo!

While testing it we noticed a few things. When trying to use the default "Matching" parameters printed in help as follows:

```
Matching:
  -c ID, --threshold ID
                        sequence identity threshold (default=0.95)
  -f FAMILY_THRESHOLD, --family_threshold FAMILY_THRESHOLD
                        protein family sequence identity threshold (default=0.7)
  --len_dif_percent LEN_DIF_PERCENT
                        length difference cutoff (default=0.98)
  --merge_paralogs      don't split paralogs
```

in fact we got the clustering done with --threshold 0.98 

```
Program: CD-HIT, V4.8.1 (+OpenMP), Oct 26 2019, 14:51:47
Command: cd-hit -T 30 -i panaroo_output/combined_protein_CDS.fasta
         -o panaroo_output/combined_protein_cdhit_out.txt
         -c 0.98 -s 0.98 -aL 0.0 -AL 99999999 -aS 0.0 -AS
         99999999 -M 0 -d 999 -g 1 -n 2
```

However in the docs you have something quite opposite:

![image](https://user-images.githubusercontent.com/23166670/108373702-785f2580-7200-11eb-84f5-81e5d95fccaf.png)

By the way, does the mode of the cleaning changes clustering parameters? We were running panaroo in ```--clean-mode sensitive```.

Thanks for taking a look at that.
Przemek

**Comment by gtonkinhill** [comment]
Hi,

Thanks for picking this up. 

We experimented with a number of different thresholds initially and I failed to update the docs correctly. Both the initial clustering threshold and length difference are set at 0.98. I've now updated the help message and docs to reflect this in dc8a961cfef8cbf5d5dc817beb0d1eea2486ef04.

The mode of cleaning does not change these parameters. It only adjusts the parameters used to decide when to delete potentially erroneous clusters.
